modules/Bio/EnsEMBL/DBSQL/AttributeAdaptor.pm

```diff
@@ -576,6 +604,22 @@ sub fetch_all_by_Translation {
 
 }
 
+sub fetch_all_by_RNAProduct {
```

Comment:
I would rather not touch any of these three, they are all thin wrappers around respective by_Object methods so either my PODs would just direct the users at non-existent documentation for said methods or I would have to decode how those methods work.

Reply:
I'm  inclined to agree with @mkszuba - although the lack of POD is less than ideal, this lack is a systemic and wide ranging issue that is best resolved by a separate documentation "sprint" rather than in this set of changes.

Reply:
No problem, thanks for looking into it though